docs/package.json

```diff
@@ -19,6 +20,10 @@
   },
   "devDependencies": {
     "@types/react": "18.2.8",
-    "@types/react-dom": "18.2.4"
+    "@types/react-dom": "18.2.4",
+    "autoprefixer": "10.4.14",
+    "postcss": "8.4.22",
+    "tailwind-merge": "^1.13.2",
```

Comment:
This is a prod dependency, although I can't find where you're using it so should probably just be removed